<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="T1_Consult_Print_Audit_Table_but">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Print Audit EI Summary Table</string>
   </property>
  </widget>
  <widget class="QTreeWidget" name="T1_Consult_Audit_EI_table">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1181</width>
     <height>211</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Batch_No</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Sub_Batch_No</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Consultancy</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Submission_Date</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Feature_No</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Status</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Type</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Letter</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Remarks</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="T1_Consult_home_but">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
  </widget>
  <widget class="QPushButton" name="T1_Consult_Order_but">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Order</string>
   </property>
  </widget>
  <widget class="QPushButton" name="T1_Consult_Update_Audit_Table_but">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>230</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QPushButton" name="T1_Consult_Minutes_but">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>260</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Minutes Preparation</string>
   </property>
  </widget>
  <widget class="QPushButton" name="T1_Consult_CheckEI_but">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>260</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Check EI</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
